dockers/docker-dhcp-relay/cli/config/plugins/dhcp_relay.py

```diff
@@ -163,14 +163,25 @@ def dhcp_relay_ipv4_helper():
 @click.argument("dhcp_relay_helpers", nargs=-1, required=True)
 @clicommon.pass_db
 def add_dhcp_relay_ipv4_helper(db, vid, dhcp_relay_helpers):
+    if is_dhcp_server_enabled(db):
```

Comment:
should we mask dhcprelay related information from show command as well? such show vlan brief

Reply:
Added